File: MODUL-2/SOAL 3/PRAK203-2410817120017-RIKA-FAULIANA-RAHMI.c
#include <stdio.h>

int main() {
    float a, b, i, j, x, y;

    // Membaca input dari pengguna
    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);

    // Melakukan perhitungan
    float hasil = (((a - b) * i) / j) - (x + y);

    // Menampilkan hasil dengan format 3 desimal
    printf("%.3f\n", hasil);

    return 0;
}

File: MODUL-2/SOAL 3/PRAK203-2410817120017-RIKA-FAULIANA-RAHMI.py
angka = []
while True:
    line = input()
    if not line:
        break
    angka.extend(line.split())

angka = list(map(float, angka))

a, b, i, j, x, y = angka

hasil = (((a - b) * i) / j) - (x + y)
print(f"{hasil:.3f}")